Sources/InstantSearchCore/CurrentFilters/Connector/CurrentFiltersConnector.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CurrentFiltersConnector.swift
+//  InstantSearchCore
+//
+//  Created by Vladislav Fitc on 29/11/2019.
+//  Copyright © 2019 Algolia. All rights reserved.
+//
+
+import Foundation
+
+/// Component that displays the current refinements and let users remove them.
+public class CurrentFiltersConnector {
```

Comment:
Small suggestion: In case you want to follow what Mouaad did, he put a link the public documentation like [this](https://github.com/algolia/instantsearch-android/pull/213/files#diff-2ea092b948d24de3d89241414cafd84bR9). Of course the danger is that the link becomes outdated at some point in the future. I'll let you decide/ chat with Mouaad if you want

Reply:
Done ✅